src/backend/ticket_service/ticket_service.py

```diff
@@ -376,4 +377,253 @@ async def delete_ticket(ticket_id: str, customer_guid: str):
         raise HTTPException(
             status_code=HTTPStatus.INTERNAL_SERVER_ERROR,
             detail="An unexpected error occurred while deleting the ticket."
+        )
+
+
+
+#Comments APIS
+
+class Comment(BaseModel):
+    comment_id:str
+    ticket_id:str
+    posted_by:str
+    comment:str
+    is_edited: bool
+    created_at:datetime
+
+class CommentRequest(BaseModel):
+    customer_guid:str
+    ticket_id:str
+    posted_by:str
+    comment:str
+
+class CommentResponse(BaseModel):
+    comment_id: str
+    status: str
+
+class CommentByTicketId(BaseModel):
+    comment_id: str
+    comment: str
+    created_at: datetime
+
+class CommentUpdate(BaseModel):
+    comment: str
+
+class CommentUpdateResponse(BaseModel):
+    comment_id: str
+    comment: str
+    is_edited: bool
+    updated_at: datetime
+
+@app.post("/comments", response_model=CommentResponse, status_code=HTTPStatus.CREATED, tags=["Comment Management"])
+async def create_comment(comment: CommentRequest):
+    """Create a new comment for a ticket"""
+    logger.debug(f"Received comment data: {comment}")
+
+    try:
+        # Call the database method to create the comment
+        logger.debug("Calling DBManager.create_comment")
+        db_response = db_manager.create_comment(
+            comment.customer_guid,
+            comment.ticket_id,
+            comment.posted_by,
+            comment.comment
+        )
+
+        logger.debug(f"Database response: {db_response}")
+        return CommentResponse(comment_id=db_response["comment_id"], status="created")
```

Comment:
Return the comment_id and the comment
Don't return the status field

Reply:
Modified Sir